Gopkg.toml

```diff
@@ -35,8 +35,7 @@
 
 [[constraint]]
   name = "github.com/shirou/gopsutil"
-  branch = "master"
-  revision = "4254ce2081c0d7994dae589a4550a4ca659ebf94"
+  revision = "v2.18.10"
```

Comment:
Why did you change this? The specified commit contains a very important fix for the Windows CPU measurement.
https://github.com/shirou/gopsutil/compare/v2.18.10...4254ce2081c0d7994dae589a4550a4ca659ebf94

Reply:
true, but it seems the current syntax is not even valid at all

Reply:
ok, I got it. 
> only one of "branch", "version" or "revision" can be specified

@troian can you please remove `branch = "master" `